packages/components/src/DatePicker/DatePicker.tsx

```diff
@@ -305,17 +306,25 @@ export const DatePicker = ({
       </div>
 
       {isOpen && (
-        <CalendarPopover referenceElement={containerRef.current}>
-          <CalendarSingle
-            id={calendarId}
-            selected={selectedDay}
-            defaultMonth={defaultMonth}
-            weekStartsOn={weekStartsOn}
-            disabled={disabledDays}
-            locale={locale}
-            onDayClick={handleCalendarDayChange}
-            onMount={handleCalendarMount}
-          />
+        <CalendarPopover
+          referenceElement={containerRef.current}
+          aria-labelledby={`${id}-calendar-label ${id}-input-label`}
+        >
+          <>
+            <VisuallyHidden id={`${id}-calendar-label`}>
+              Choose date for:
```

Comment:
Small suggestion to give a bit more info that it's a selection from a calendar. Thoughts?

```suggestion
              Select date from calendar for:
```

Reply:
Good point, it's a nice way to delineate between the "choose date" button and the calendar